Library/ViewModels/PledgeViewModel.swift

```diff
@@ -26,9 +26,12 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       .map(first)
       .skipNil()
 
-    self.amountAndCurrency = projectAndReward.signal
+    self.amountAndCurrencyAndDeliveryDate = projectAndReward.signal
       .map { (project, reward) in
-        (reward.minimum, currencySymbol(forCountry: project.country).trimmed())
+        (reward.minimum, currencySymbol(forCountry: project.country).trimmed(),
```

Comment:
Can we format this slightly better so that it's more readable?

Maybe returning each item of the tuple per line?

```
(
  first,
  second,
  third
)
```
?